examples/manual/manual.md

```diff
@@ -0,0 +1,54 @@
+# End to end TMT tutorial
+## Setup
+1. Install tmt
+```bash
+sudo dnf install -y tmt-all
+```
+2. Create and switch to a brand new git repo
+```bash
+cd `mktemp -d`
+git init
+```
+## Test steps
+### Step 1
+Verify tmt show help page
+```bash
+tmt --help
+```
+#### Expected result
```

Comment:
For the fixed heading names I would suggest something rather shorter, ideally one word, so that writing new manual tests is as easy as possible even without copy & pasting. What about just `Result` or `Expect`?